backend/config/corpora-api.yml

```diff
@@ -461,7 +466,7 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
-
+          
```

Comment:
Morning @Bento007 ! Just wondering if we need to update this `yml` file to add `name` to `project.link[]`, so we can display each external link's name 😄 Thank you!

Reply:
We don't have a project link name yet -- the change to add the information to the ProjectLink table and the Corpora schema is underway.

Reply:
Ah got it! Thanks, @maniarathi 🙏 

Is the link name going to be available for the FE/BE integration this week? If not, for Friday's demo, we can either just leave the "More information" column blank or I can hardcode some links for the demo to look pretty ✨ 😄 

<img width="1440" alt="Screen Shot 2020-07-14 at 3 20 48 PM" src="https://user-images.githubusercontent.com/6309723/87482179-ba2e4a80-c5e5-11ea-8366-eaadaed8e872.png">

Reply:
Nope, let's skip the link name for the demo. The database changes won't be in by then.

I'd prefer for the demo to just leave the column blank instead of hardcoding :) Tryna keep the demo real :D

Reply:
Ahaha good call 🎺 ! OK I'll leave it blank 😄 👌 Thank you!